nrf_modem/include/nrf_socket.h

```diff
@@ -471,6 +471,30 @@ typedef uint32_t nrf_fd_set;
 #define NRF_SO_SILENCE_IP_ECHO_REPLY   31
 #define NRF_SO_SILENCE_IPV6_ECHO_REPLY 32
 #define NRF_SO_REUSEADDR	       40
+/** Release Assistance Indication feature: This will indicate that the next call to send/sendto will
+ * be the last one for some time. Which means that the modem can bring down the network quicker
+ * when this data is sent.
```

Comment:
```suggestion
 * be the last one for some time, which means that the modem can get out of connected mode quicker
 * when this data is sent.
```

Reply:
It would be better to talk about "connected mode". Modem is not disconnected from the network, but the RRC connection is released and no user data transfer is possible in idle mode.